src/pages/jiyulBaek/Main/Main.js

```diff
@@ -1,7 +1,293 @@
+                src="/images/jiyulBaek/bookmark.png"
+                alt="bookmark-icon"
+              />
+              <span>저장됨</span>
+            </li>
+            <li className="nav-profile-menu">
+              <img
+                className="fa-gear"
+                src="/images/jiyulBaek/gear.png"
+                alt="setting-icon"
+              />
+              <span>설정</span>
+            </li>
+            <li className="nav-profile-menu">로그아웃</li>
+          </ul>
+        </div> */}
+      </header>
+
+      <main>
+        <section className="feeds">
+          <div className="top">
+            <div className="top-logo">
+              <img
+                src="https://images.unsplash.com/photo-1507166763745-bfe008fbb831?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"
+                alt="accountImage1"
+              />
+              <span>jy_baek</span>
+            </div>
+            <div className="top-icon">
+              <img
+                className="fa-ellipsis"
+                src="/images/jiyulBaek/ellipsis.png"
+                alt="ellipsis"
+              />
+            </div>
+          </div>
+
+          <div className="image">
+            <img src="/images/jiyulBaek/feed.jpg" alt="feedImage" />
+          </div>
+
+          <div className="feed-icons">
+            <div className="feed-icons-left">
+              <img
+                className="fa-heart"
+                src="/images/jiyulBaek/heart_white.png"
+                alt="unclicked_heart"
+              />
+              <img
+                className="fa-comment"
+                src="/images/jiyulBaek/comment.png"
+                alt="comment-icon"
+              />
+              <img
+                className="fa-arrow-up-from-bracket"
+                src="/images/jiyulBaek/ArrowUpFromBracket.png"
+                alt="share-icon"
+              />
+            </div>
+            <div className="feed-icons-right">
+              <img
+                className="fa-bookmark"
+                src="/images/jiyulBaek/bookmark.png"
+                alt="bookmark-icon"
+              />
+            </div>
+          </div>
+
+          <div className="likes">
+            <img
+              src="https://images.unsplash.com/photo-1587161584760-f51779fb276a?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"
+              alt="accountImage2"
+            />
+            <span>
+              <span className="likesInfo">aineworld</span>님 외
+              <span className="likesInfo">10명</span>이 좋아합니다.
+            </span>
+          </div>
+
+          <div className="contents">
+            <span className="contents-account">jy_baek</span>
+            <span>위워크에서 진행된 베이킹 클래스...</span>
+            <span>더 보기</span>
+            <div>42분 전</div>
+          </div>
+
+          <ul className="comment-list">{commentList}</ul>
```

Comment:
```suggestion
          <ul className="comment-list">{comments.map(comment => (
    <Comment key={comment.id} comment={comment.comment} />
  ))}</ul>
```
위의 리뷰를 토대로 보면 이렇게가 되겠네요!